### PR TITLE
add instructions to upgrade admission-control images

### PR DESCRIPTION
**Pull request by fredrb**
Adds instructions to update admission-control images under `Update Other Images`

### CODE REVIEW

File: modules/update-other-images-40.adoc [hunk 1]
```diff
@@ -79,6 +79,11 @@ Or if you're using collector slim:
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:{rhacs-version}
 ----
+. Update the Admission Control image:
++
+----
+$ oc -n stackrox set image deploy/admission-control admission-control=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version} <1>
```
Comment (msugakov) [inline modules/update-other-images-40.adoc]: image
```suggestion
$ oc -n stackrox set image deploy/admission-control admission-control=registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
```

File: modules/update-other-images-40.adoc [hunk 2]
```diff
@@ -79,6 +79,11 @@ Or if you're using collector slim:
 ----
 $ oc -n stackrox set image ds/collector collector=registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:{rhacs-version}
 ----
+. Update the Admission Control image:
```
Comment (msugakov) [inline modules/update-other-images-40.adoc:82]: I _think_ that, when deploying, you can opt out from deploying admission control. In this case the command will fail.
I'm checking that now.

**Review by msugakov — COMMENTED** [review]
Thanks, this is what I meant here https://github.com/stackrox/openshift-docs/pull/8#discussion_r794666912

File: modules/update-other-images-40.adoc (hunk 1 above)
Comment (fredrb) [reply modules/update-other-images-40.adoc]: Again 😵‍💫 
Sorry about that

File: modules/update-other-images-40.adoc (hunk 2 above)
Comment (msugakov) [reply modules/update-other-images-40.adoc:82]: Never mind. I saw that you can disable something that looked like you disable admission controller but I still found that the resulting manifests do deploy admission controller.


Your change is correct per my undertanding, please carry on!

**Review by msugakov — APPROVED** [review]

**Review by misberner — APPROVED** [review]

**Review by vikin91 — APPROVED** [review]

**Pull request merged**